components/buttongroup/buttons.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Buttons
+page_title: ButtonGroup Buttons
+description: Explore the available buttons in the ButtonGroup for Blazor.
+slug: buttongroup-buttons
+tags: telerik,blazor,toggle,button,group,type
+published: True
+position: 2
+---
+
+# ButtonGroup Buttons
+
+The ButtonGroup component supports two types of buttons that have different behavior:
+
+* [`ButtonGroupToggleButton`](#buttongroup-togglebutton)
+* [`ButtonGroupButton`](#buttongroup-button)
+
+You can add the desired button instances by declaring the dedicated button tags. Additionally, you can individually confire their [appearance]({%slug buttongroup-appearance%}), [enabled/disabled state](#disabled-state) and [visibility](#visibility) through the paramaters each button tag exposes.
+
+## ButtonGroup ToggleButton
+
+The `ButtonGroupToggleButton` becomes primary when clicked and de-selects when another one is clicked. If multiple selection is enabled, the user can select more than one `ButtonGroupToggleButton` at a time. Clicking on a selected button in this case will de-select it. Read more in the [Selection]({%slug buttongroup-selection%}) article.
```

Comment:
"primary" is coupled with a name for a style state. A novice reader will not understand this. If you wish to still mention the primary style, link to somewhere with more information.

```suggestion
The `ButtonGroupToggleButton` becomes selected when clicked and de-selects when another one is clicked. If multiple selection is enabled, the user can select more than one `ButtonGroupToggleButton` at a time. Clicking on a selected button in this case will de-select it. Read more in the [Selection]({%slug buttongroup-selection%}) article.
```